webpack.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path')
+const webpack = require('webpack')
+const webpackConfig = require('@nextcloud/webpack-vue-config')
+
+webpackConfig.entry = {
+	text: path.join(__dirname, 'src', 'main.js'),
+	files: path.join(__dirname, 'src', 'files.js'),
+	public: path.join(__dirname, 'src', 'public.js'),
+	viewer: path.join(__dirname, 'src', 'viewer.js'),
+}
+
+webpackConfig.output.chunkFilename = '[id].js?v=[contenthash]'
```

Comment:
Any reason?

Reply:
Mainly to keep chunk names persistent with the `highlight/{lang}.js` ones, but I'm also fine to remove that.

Reply:
The idea is to have all apps preceding their appName on files names, so it's easier to check what is what on network logging and such :wink: 
I'm always happier when we stick to the origal config files for less maintenance, but it's personal opinion, so I'm good with whatever you prefer :wink: